<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogImportWizard</class>
 <widget class="QDialog" name="DialogImportWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>680</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Animation Sheet Import Wizard</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stkTabStack">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabCropper">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,1,0,4">
         <item>
          <widget class="QGroupBox" name="grpInfo">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="title">
            <string>Info</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="lblCurImage">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Current Image: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblSprite">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Target Sprite: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="grpTools">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>256</width>
             <height>32</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="3" column="0">
             <widget class="QPushButton" name="btnSaveTemplate">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Save Template</string>
              </property>
              <property name="icon">
               <iconset resource="../../../appres.qrc">
                <normaloff>:/resources/fugue-icons-3.5.6/icons/blueprint--arrow.png</normaloff>:/resources/fugue-icons-3.5.6/icons/blueprint--arrow.png</iconset>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QPushButton" name="btnLoadTemplate">
              <property name="text">
               <string>Load Template</string>
              </property>
              <property name="icon">
               <iconset resource="../../../appres.qrc">
                <normaloff>:/resources/fugue-icons-3.5.6/icons/blueprint.png</normaloff>:/resources/fugue-icons-3.5.6/icons/blueprint.png</iconset>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="btnOpenImage">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>Open Image..</string>
              </property>
              <property name="icon">
               <iconset resource="../../../appres.qrc">
                <normaloff>:/resources/fugue-icons-3.5.6/icons/image-import.png</normaloff>:/resources/fugue-icons-3.5.6/icons/image-import.png</iconset>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="btnImport">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Finalize Import</string>
              </property>
              <property name="icon">
               <iconset resource="../../../appres.qrc">
                <normaloff>:/resources/fugue-icons-3.5.6/icons/gear.png</normaloff>:/resources/fugue-icons-3.5.6/icons/gear.png</iconset>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="btnReset">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Reset</string>
              </property>
              <property name="icon">
               <iconset resource="../../../appres.qrc">
                <normaloff>:/resources/fugue-icons-3.5.6/icons/eraser.png</normaloff>:/resources/fugue-icons-3.5.6/icons/eraser.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="grpOptions">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="title">
            <string>Options</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCheckBox" name="chkReplacePalette">
              <property name="text">
               <string>Replace palette</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="tblvProperties">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tblProperties">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>256</width>
             <height>0</height>
            </size>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectItems</enum>
           </property>
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <property name="columnCount">
            <number>1</number>
           </property>
           <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>24</number>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>24</number>
           </attribute>
           <attribute name="verticalHeaderShowSortIndicator" stdset="0">
            <bool>true</bool>
           </attribute>
           <row>
            <property name="text">
             <string>Cell Width</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Cell Height</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Cell HMargin</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Cell VMargin</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string comment="Number of horizontal animation cells for a single animation sequence.">Nb HCells</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Nb VCells</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Nb HSequences</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Nb VSequences</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>X</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Y</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Discard Empty Cells</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Default Ticks Per Frames</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Values</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string>32</string>
            </property>
           </item>
           <item row="1" column="0">
            <property name="text">
             <string>32</string>
            </property>
           </item>
           <item row="2" column="0">
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item row="3" column="0">
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item row="4" column="0">
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item row="5" column="0">
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item row="6" column="0">
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item row="7" column="0">
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item row="8" column="0">
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item row="9" column="0">
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item row="10" column="0">
            <property name="text">
             <string>true</string>
            </property>
           </item>
           <item row="11" column="0">
            <property name="text">
             <string>1</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGraphicsView" name="gvSheetEditor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>256</width>
           <height>256</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="backgroundBrush">
          <brush brushstyle="SolidPattern">
           <color alpha="0">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
         <property name="renderHints">
          <set>QPainter::LosslessImageRendering</set>
         </property>
         <property name="dragMode">
          <enum>QGraphicsView::ScrollHandDrag</enum>
         </property>
         <property name="cacheMode">
          <set>QGraphicsView::CacheBackground</set>
         </property>
         <property name="resizeAnchor">
          <enum>QGraphicsView::AnchorUnderMouse</enum>
         </property>
         <property name="viewportUpdateMode">
          <enum>QGraphicsView::FullViewportUpdate</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../appres.qrc"/>
 </resources>
 <connections/>
</ui>
